RailwayCo/Assets/Scripts/GameLogic/Cargo/CargoMaster.cs

```diff
@@ -5,14 +5,14 @@
 public class CargoMaster : IPlayfab
 {
     private WorkerDictHelper<Cargo> Collection { get; set; }
-    private WorkerDictHelper<CargoModel> CargoCatalog { get; set; }
+    private WorkerDictHelper<CargoModel> Catalog { get; set; }
```

Comment:
`private set` to be explicit?
and then the naming convention according to access class

```
private int _privVar
public int PubVar
```

Reply:
See above for `private set`

Oh, actually that [naming convention](https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions#camel-case) applies to fields and not properties.